core/src/loader.rs

```diff
@@ -1540,6 +1541,40 @@ impl<'gc> Loader<'gc> {
 
                 let mut activation = Avm2Activation::from_nothing(uc);
 
+                fn strip_bom<'a, 'gc: 'a>(
+                    activation: &mut Avm2Activation<'a, 'gc>,
```

Comment:
We should probably avoid duplicating this function across here and `byte_array`- could you define it in `byte_array.rs`?

Reply:
sure